src/OrchardCore.Modules/OrchardCore.Admin/Views/AdminDashboard.cshtml

```diff
@@ -1,4 +1,23 @@
-<div class="alert alert-success">
-    <h4>@T["Welcome to Orchard"]</h4>
-    <p>@T["Feel free to browse the menu and discover all its possibilities."]</p>
+@using OrchardCore.DisplayManagement;
+@inject OrchardCore.DisplayManagement.Descriptors.IShapeTableManager ShapeTableManager
+
+@{
+    var shapeTable = ShapeTableManager.GetShapeTable("TheAdmin");
```

Comment:
I would suggest this is done with Drivers, and a 
`IDisplayManager<AdminDashboard>()` (making an identifier class called `AdminDashboard` in `Admin.Abstractions` to drive the display with)

Here this code will fail if `GetShapeTable("TheAdmin")` a different admin theme was used.

If you do it with drivers, then you get placement - the ability to hide them / move them around. All the cool things that modular display management does.